api/src/pcapi/core/educational/models.py

```diff
@@ -1203,6 +1203,8 @@ class CollectiveOfferRequest(PcObject, Base, Model):
 
     comment: str = sa.Column(sa.Text, nullable=False)
 
+    dateCreated: date = sa.Column(sa.Date, nullable=True, server_default=sa.func.current_date())
```

Comment:
(détail) As-tu un ticket (ou, à défaut, ajouter un FIXME ici) pour rendre cette colonne `NOT NULL` ?

(Idem pour `GetCollectiveOfferRequestResponseModel.dateCreated`.)

Reply:
Précision : je me dis qu'on pourra remplir la table avec des valeurs "bidon" mais relativement proches de la réalité, genre 2023-06-01T00:00:00. (Enfin, si on a des données en production avant l'ajout de cette colonne. Pour l'instant, la table est vide.)